project-koku/koku
# COST-1424: Correcting GB-mo daily usage calculation

The GB-Mo SQL was effectively calculating GB-day for daily usage values.  Adding the missing () ensures that we are dividing by the number of days in the month to get GB-mo for daily usage.

**Testing**
1. Add OCP source with $1 GB-mo storage_gb_usage_per_month cost model.
2. Ingest OCP data with 1GB of PVC usage for the month.
3. Verify that Volume usage is correct.
4. Verify volume cost is correct

**Test Results**
[cost_1424_ut.txt](https://github.com/project-koku/koku/files/6658243/cost_1424_ut.txt)
